docs/Guides/Games/Sonic1-2/Datapack.md

```diff
@@ -28,18 +28,28 @@ The datapacks for Sonic 1 and 2 are required for both playing and modding the ga
 ## Obtaining the Datapacks
 === "Android"
 
-    TODO: Verify that this still works, I basically just copy-pasted my old tutorial from three years ago, also needs screenshots
+    TODO: {--Verify that this still works, I basically just copy-pasted my old tutorial from three years ago, also needs screenshots--} Verified by Ivan951236
+	
+    ![1.jpeg](https://raw.githubusercontent.com/Ivan951236/RSDK-Modding-Wiki/refs/heads/patch-1/docs/Guides/Games/Sonic1-2/1.jpg?raw=true)
 
     Use your APK extracting app of choice to get an APK file of Sonic 1 or Sonic 2.
+	
+    ![1.jpeg](https://raw.githubusercontent.com/Ivan951236/RSDK-Modding-Wiki/refs/heads/patch-1/docs/Guides/Games/Sonic1-2/1.jpg?raw=true)
```

Comment:
Please use a local path relative to the `docs` directory instead of linking GitHub user content. Refer to the other images in this page for how to do this. Please also add proper alt text to the images. This applies to all the images you added.